BankManagerConsoleApp/Model/Console/Menu.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Console.swift
+//  BankManagerConsoleApp
+//
+//  Created by 우롱차, 민성 on 2022/04/28.
+//
+
+import Foundation
+
+struct Menu {
+    private enum MenuContstant {
+        static let menual =
+        """
+        1 : 은행개점
+        2 : 종료
+        """
+        static let InputMessage = "입력: "
+        static let openBank = "1"
+        static let finsh = "2"
+    }
+
+    func start() {
+        var inputValue: String
+
+        print(MenuContstant.menual)
+        inputValue = readInput()
+
+        switch inputValue {
+        case MenuContstant.openBank:
+            startWorking()
+            start()
+        case MenuContstant.finsh:
+            return
+        default:
+            start()
+        }
+    }
+
+    private func readInput() -> String {
+        print(MenuContstant.InputMessage, terminator: "")
+        guard let input = readLine() else {
+            return ""
+        }
+
+        return input
+    }
```

Comment:
사용성 확장을 위해 "입력:" 프린트문을 분리하여 연산 프로퍼티로 입력된 값을 반환하는 연산 프로퍼티로 선언하는건 어떨까요?
```suggestion
    private var readInput: String {
        guard let input = readLine() else {
            return ""
        }
        return input
    }
```

Reply:
이 방식으로 수정해보록 하겠습니다.